index.js

```diff
@@ -4,9 +4,9 @@ var exports = module.exports = { hoodieApi: hoodieApi }
 
 var EventEmitter = require('events').EventEmitter
 
-function hoodieApi () {
+function hoodieApi (emitter) {
```

Comment:
I was wondering about that as well and agree with adding it to an options argument. I'll do the same with https://github.com/hoodiehq/pouchdb-hoodie-sync/pull/42 .